server/src/server.ts

```diff
@@ -42,6 +42,7 @@ let projectsFiles: Map<
     openFiles: Set<string>;
     filesWithDiagnostics: Set<string>;
     bsbWatcherByEditor: null | ChildProcess;
+    hasPromptedToStartBuild: boolean;
```

Comment:
Not about this PR per se.
Can we be super-explicit that this whole area is the server's global state?